client/components/jetpack/daily-backup-status/test/action-buttons.jsx

```diff
@@ -109,4 +109,22 @@ describe( 'ActionButtons', () => {
 			rewind_id: rewindId,
 		} );
 	} );
+
+	test( 'enables clone button', async () => {
+		const user = userEvent.setup();
+		const rewindId = 'test';
+		const onClickClone = jest.fn();
+		render(
+			<ActionButtons
+				rewindId={ rewindId }
+				availableActions={ [ 'clone' ] }
+				onClickClone={ onClickClone }
```

Comment:
Need to pass the rewindId to `onClickClone( rewindId )` to get it back to the clone-flow handler.